1_antenatal_care.do

```diff
@@ -157,21 +158,7 @@
 				replace c_anc_ski = . if `var' == "?"  // missing for DK/missing if no other skilled provider used
 			}	
 			replace c_anc_ski = . if bl2 != 1 | ~inrange(wb4,15,49)			// missing for births > 24 months ago
-		}
```

Comment:
Could you confirm why moving sections for Tonga2019? Is this a rebase issue? @robin-wang

Reply:
It's intended. 
Starting with line150, we have a section that's with ~Georgia, this is identical to the Tonga specific chunk. Hence the removal.

Reply:
Similar treatment for bp\bs\ur in the anc section.